### ISSUE TITLE
Add Animal Thumbnails to Manager Views

### ISSUE THREAD

**Issue by markottaviani** [ophrescue/RescueRails#1605]
> . Presently as an AC it's tedious to click on each dog name to see info/picture about a dog.  Is there a way to add a thumbnail picture which will at least let us know what the dog looks like. If I have an adopter interested in a large male and I see a little dog pic I'd be able to skip through quicker. Anything to help us save time.
Thanks.

Same feature should be added for cats as well.  

May be some issues with performance in finding the primary photo for a given animal quickly.

**Comment by danisaurio** [comment]
@markottaviani I assume this Thumbnail view should be added in the manager view of dogs/cats... is that correct??
Besides the picture, what else is useful enough to be added to the thumbnail?
I was thinking about editing how the information is displayed instead of doing a thumbnail, would that work?

**Comment by markottaviani** [comment]
Hi Daniela,  Yes, thumbnail should be added to to existing Index manger view for dogs and cats. Image can be added on the left as in this screenshot: https://share.getcloudapp.com/YEupnKlv

You're welcome to provide some screenshots if you'd like to make other suggestions, but for this particular feature, it is just to add a thumbnail to the existing view, thanks!

**Comment by danisaurio** [comment]
@markottaviani 
Hey Mark, I discovered that there are some flags icons that are not displayed. Should I make them visible?
This is what I've done so far
PS. I uploaded the picture of the 1st entry, those are my pets lol
![image](https://user-images.githubusercontent.com/41554471/87989881-b001c580-ca97-11ea-91ff-3cbff10fc321.png)

**Comment by markottaviani** [comment]
Thank you for the screenshot.  Can you please limit the change to just adding the thumbnail?  Image can be added on the left as in this screenshot: https://share.getcloudapp.com/YEupnKlv  Thanks!

**Comment by danisaurio** [comment]
Of course, no problem!
The thing is that I changed the [_dog.html.erb](https://github.com/ophrescue/RescueRails/blob/master/app/views/dogs/manager/_dog.html.erb) file bc the table wasn't actually an HTML table element, it was a div with classes "row" or "col" and some offsets from bootstrap, which can be really confusing in terms of accessibility. Besides, when adding the thumbnail where you suggest, made the display a bit messed up, which would require some CSS to position as expected.

For these reasons, I decided to reimplement it with a table according to Bootstrap 4. Nevertheless, I can make a pull request keeping the original structure if that makes sense :)

Attaching screenshot of an image with the original structure
![image](https://user-images.githubusercontent.com/41554471/88080999-ad08e280-cb34-11ea-8d21-3a776415722e.png)
